apps/web/pages/api/orgMigration/moveTeamToOrg.ts

```diff
@@ -0,0 +1,74 @@
+import { getFormSchema } from "@pages/settings/admin/orgMigrations/moveTeamToOrg";
+import { getSession } from "next-auth/react";
+import type { NextApiRequest, NextApiResponse } from "next/types";
+
+import { HttpError } from "@calcom/lib/http-error";
+import logger from "@calcom/lib/logger";
+import { safeStringify } from "@calcom/lib/safeStringify";
+import { getTranslation } from "@calcom/lib/server";
+import { UserPermissionRole } from "@calcom/prisma/enums";
+
+import { moveTeamToOrg } from "../../../lib/orgMigration";
+
+const log = logger.getSubLogger({ prefix: ["moveTeamToOrg"] });
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const rawBody = req.body;
+
+  log.debug(
+    "Moving team to org:",
+    safeStringify({
+      body: rawBody,
+    })
+  );
+
+  const translate = await getTranslation("en", "common");
+  const moveTeamToOrgSchema = getFormSchema(translate);
+
+  const parsedBody = moveTeamToOrgSchema.safeParse(rawBody);
+
+  // Don't know why but if I let it go to getSession, it doesn't return the session.  🤯
+  req.body = null;
```

Comment:
```suggestion
delete req.body 
```
NIT: 
Might be a bit cleaner instead of setting a null value